test/LaunchDarkly.ServerSdk.Redis.Tests/RedisDataStoreTest.cs

```diff
@@ -26,13 +27,27 @@ private IPersistentDataStoreFactory MakeStoreFactory(string prefix)
             return Redis.DataStore().Prefix(prefix);
         }
 
-        private Task ClearAllData(string prefix)
+        private async Task ClearAllData(string prefix)
         {
-            using (var cxn = ConnectionMultiplexer.Connect("localhost:6379,allowAdmin=true"))
-            {
-                cxn.GetServer("localhost:6379").FlushDatabase();

```

Comment:
`FlushDatabase` was not the right way to get a clean test state, because it would remove _all_ data from Redis, including data with other prefixes.